aten/src/ATen/native/MaxPooling.cpp

```diff
@@ -74,7 +74,7 @@ static void check_max_pool1d(
   TORCH_CHECK(
       dilation[0] > 0, "max_pool1d() dilation must be greater than zero, but got ", dilation[0]);
 
-  const int64_t OW = pooling_output_shape(self.size(-1), kernel_size[0], padding[0], stride[0], dilation[0], ceil_mode);
+  const int64_t OW = pooling_output_shape(self.sym_size(-1).guard_int(__FILE__, __LINE__), kernel_size[0], padding[0], stride[0], dilation[0], ceil_mode);
```

Comment:
Don't see a way to make `pooling_output_shape` purely symbolic, so used a guard.